From 19e161093670cfa2bfb7142efac69946975bafce Mon Sep 17 00:00:00 2001
From: Aishwarya R <quic_aisr@quicinc.com>
Date: Wed, 12 Jul 2023 16:24:15 +0530
Subject: [PATCH] ath12k: Set scan priority to medium

 During OBSS collision detection firmware sets the scan
 priority to medium. As ath12k scan priority is low, OBSS scan results are
 getting aborted due to priority mismatch. As per the firmware recommendation
 setting the priority to medium.

Signed-off-by: Aishwarya R <quic_aisr@quicinc.com>
---
 drivers/net/wireless/ath/ath12k/wmi.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/ath/ath12k/wmi.c b/drivers/net/wireless/ath/ath12k/wmi.c
index ca71791..9f5742a 100644
--- a/drivers/net/wireless/ath/ath12k/wmi.c
+++ b/drivers/net/wireless/ath/ath12k/wmi.c
@@ -2852,7 +2852,7 @@ void ath12k_wmi_start_scan_init(struct ath12k *ar,
 {
 	/* setup commonly used values */
 	arg->scan_req_id = 1;
-	arg->scan_priority = WMI_SCAN_PRIORITY_LOW;
+	arg->scan_priority = WMI_SCAN_PRIORITY_MEDIUM;
 	arg->dwell_time_active = 50;
 	arg->dwell_time_active_2g = 0;
 	arg->dwell_time_passive = 150;
-- 
2.17.1

